### PR TITLE
Changed Examples for Pep8 Compliance

### PR DESCRIPTION
**Pull request by sirlittle**
Changed demo_parasite_axes2.py and demo_floating_axis.py to add pep8 compliance as well a docstring for the sphinx_gallarey.

### CODE REVIEW

File: examples/axisartist/demo_parasite_axes2.py [hunk 1]
```diff
@@ -1,45 +1,52 @@
+"""
+Parasite Axis Demo
+
+The following code is an example of of a parasite axis.
+It aims to show a user how to plot multiple different values
+onto one single plot. Notice how in this example, par1 and
+par2 are both calling twinx meaning both are tied directly to
+the x-axis. From there, each of those two axis can behave 
+seperately from the current x-axis as well as the other
```
Comment (NelleV) [inline examples/axisartist/demo_parasite_axes2.py]: I think there is a type here: seperately -> separately.

**Comment by tacaswell** [comment]
Looks like there is some trailing whitespace left...

File: examples/axisartist/demo_floating_axis.py [hunk 2]
```diff
@@ -1,30 +1,35 @@
 """
-An experimental support for curvilinear grid.
+Axis Within Rectangular Frame
+
+The following code demonstrates how to put a floating
+polar curve within a rectangular box. In order to get
+a better sense of polar curbes, please look at
```
Comment (QuLogic) [inline examples/axisartist/demo_floating_axis.py]: curbes -> curves

File: examples/axisartist/demo_parasite_axes2.py [hunk 3]
```diff
@@ -1,45 +1,52 @@
+"""
+Parasite Axis Demo
+
+The following code is an example of of a parasite axis.
```
Comment (QuLogic) [inline examples/axisartist/demo_parasite_axes2.py]: Duplicate "of".

File: examples/axisartist/demo_parasite_axes2.py (hunk 1 above)
Comment (QuLogic) [inline examples/axisartist/demo_parasite_axes2.py]: I don't quite understand. The previous sentence says that both are tied to the x-axis, but this sentence says they can behave separately.

File: examples/axisartist/demo_floating_axis.py [hunk 4]
```diff
@@ -1,30 +1,35 @@
 """
-An experimental support for curvilinear grid.
+Axis Within Rectangular Frame
```
Comment (TrishGillett) [inline examples/axisartist/demo_floating_axis.py]: @NelleV Do we want to capitalize every word in the sphinx-gallery docstring title, or only the first word?  I've been doing only the first word like in the example [here](http://matplotlib.org/devdocs/devel/MEP/MEP12.html).  I know this seems nitpicky, but a style mismatch will stand out in the sphinx gallery view. :P

Comment (NelleV) [inline examples/axisartist/demo_floating_axis.py]: First word sounds better.
Sorry @sirlittle : do you mind quickly updating this?

**Comment by QuLogic** [comment]
Oops, missed the whitespace stuff, fixed in 023caeceeac8f1d6748dc08002d2c883d2557748.

**Comment by QuLogic** [comment]
Backported to v2.x as 4833e41de7e70713bc573a2aeb33519497a8db92 and 0da61ef63202c0335834586b266f7b539fbfcbfa.

**Pull request merged**